.eslintrc.js

```diff
@@ -323,6 +323,8 @@ const rules = {
   'react/self-closing-comp': 1,
   'react/sort-comp': 0,
   'react/sort-prop-types': 0,
+  'react/state-in-constructor': 0,
+  'react/static-property-placement': 0,
```

Comment:
Does this mean we don't want to force the static definition in a class at the top? If so, maybe we should change to 1?

Reply:
Enabled now.